docs/dev.rst

```diff
@@ -37,6 +97,147 @@ The Symbols Service webapp is at: http://localhost:3000
 The Symbolication Service webapp is at: http://localhost:8050
 
 
+Bugs / Issues
+=============
+
+All bugs are tracked in `Bugzilla <https://bugzilla.mozilla.org/>`_.
+
+Write up a new bug:
+
+https://bugzilla.mozilla.org/enter_bug.cgi?product=Tecken&component=General
+
+If you want to do work for which there is no bug, it's best to write up a bug
+first. Maybe the ensuing conversation can save you the time and trouble
+of making changes!
+
+
+Code workflow
+=============
+
+Bugs
+----
+
+Either write up a bug or find a bug to work on.
+
+Assign the bug to yourself.
+
+Work out any questions about the problem, the approach to fix it, and any
+additional details by posting comments in the bug.
+
+
+Pull requests
+-------------
+
+Pull request summary should indicate the bug the pull request addresses. For
+example::
+
+    bug nnnnnnn: removed frob from tree class
+
+Pull request descriptions should cover at least some of the following:
+
+1. what is the issue the pull request is addressing?
+2. why does this pull request fix the issue?
+3. how should a reviewer review the pull request?
+4. what did you do to test the changes?
+5. any steps-to-reproduce for the reviewer to use to test the changes
+
+After creating a pull request, attach the pull request to the relevant bugs.
+
+We use the `rob-bugson Firefox addon
+<https://addons.mozilla.org/en-US/firefox/addon/rob-bugson/>`_. If the pull
+request has "bug nnnnnnn: ..." in the summary, then rob-bugson will see that
+and create a "Attach this PR to bug ..." link.
+
+Then ask someone to review the pull request. If you don't know who to ask, look
+at other pull requests to see who's currently reviewing things.
+
+
+Code reviews
+------------
+
+Pull requests should be reviewed before merging.
+
+Style nits should be covered by linting as much as possible.
+
+Code reviews should review the changes in the context of the rest of the system.
```

Comment:
I get the general idea here but I feel this was added with something more specific in mind. If that's the case, can we have an example?

Reply:
If you have ideas for an example, we can add it.

Reply:
Nevermind: I see this text was present before in the docs and was moved. I thought you had just written it based on a recent situation.